compiler-core/src/format/tests.rs

```diff
@@ -6156,3 +6156,19 @@ fn newlines_are_not_stripped_if_two_consecutive_anonymous_function_are_passed_as
 "#
     );
 }
+
+#[test]
+fn const_long_concat_string() {
+    assert_format_rewrite!(
+        r#"const long_string = "some" <> " very" <> " long" <> " string" <> " indeed" <> " please" <> " break"
+"#,
+        r#"const long_string = "some"
+  <> " very"
+  <> " long"
+  <> " string"
+  <> " indeed"
+  <> " please"
+  <> " break"
+"#
+    );
+}
```

Comment:
Can we have tests for unbroken concatenation expressions and also something like `const x = "some long string 1" <> "some long string 2" <> ["here is a list", "with several elements", "in order to make it be too long to fit on one line", "so we can see how it breaks", "onto multiple lines"] <> "and a last string`